docs/OV_Runtime_UG/migration_ov_2_0/deployment_migration.md

```diff
@@ -0,0 +1,156 @@
+# Install & Deployment {#openvino_2_0_deployment}
+
+## Introduction
+
+"Easy to use" is one of the main points for OpenVINO 2.0 concept. This includes not only simplification of migration from frameworks to OpenVINO Toolkit, but also how OpenVINO is organized, how the development tools are used, how to develop and deploy OpenVINO-based applications.
+
+Starting from OpenVINO 2.0, Model Optimizer, Post Training Optimization tool, Open Model Zoo tools and other Python-based Development tools are distributed via [PyPi](https://pypi.org/project/openvino-dev/) only. This simplifies user experience, because in older versions of OpenVINO development tools should be installed and used differently dependning on distribution type (OpenVINO Installer versus PyPi), but also ensures that dependencies are handled properly via `pip` package manager as well as supports virtual environments for development tools.
+
+Regarding the OpenVINO Installer, we have tried further to reorganize package structure to have it in a good shape:
+
+- `runtime` folder with OpenVINO Runtime include headers, libraries and CMake interfaces.
+- `tools` folder with [compile_tool](../../../tools/compile_tool/README.md), [deployment manager](../../install_guides/deployment-manager-tool.md), `requirement.txt` files with link to corresponding version of `openvino-dev` package.
+- `python` folder with OpenVINO Python Runtime.
```

Comment:
What is "OpenVINO Python Runtime"? I think it's a kind of incorporating the OpenVINO brand, which is not allowed. Is it "the Python version for OpenVINO Runtime"?

Reply:
the Python version for OpenVINO Runtime - yes